common/static/common/js/discussion/utils.js

```diff
@@ -386,8 +386,7 @@
                         /*
                          bug fix, ordering is off
```

Comment:
This comment was there when this file was a .coffee file. This can be verified by looking at this [commit](https://github.com/edx/edx-platform/commit/2b8c02a759c04477f1d003a86f383a860b1b85a6). In this commit file was converted from .coffee to .js and this particular comment was there at that time.